perllib/FixMyStreet/App/Controller/Root.pm

```diff
@@ -39,6 +39,7 @@ sub auto : Private {
 
     # decide which cobrand this request should use
     $c->setup_request();
+    $c->detach('/auth/redirect', []) if $c->cobrand->call_hook(check_login_disallowed => $c);
```

Comment:
You don't need to pass in `$c` here, it's available as `$self->{c}` in a cobrand during a web request.